### ISSUE TITLE
能否不用xposed、而是直接设置为默认短信应用？

### ISSUE THREAD

**Issue by sammlicke** [apsun/NekoSMS#85]
你好，我是NekoSMS的用户，之前用NekoSMS非常喜欢，功能便捷易用完美满足拦截垃圾短信的需求。
后来换了手机，没有root不能用xposed，试过各种短信app，拦截功能都有各种各样的问题，没有NekoSMS好用。
偶然读到NekoSMS的功能介绍，其中提到之所以使用xposed是为了绕过系统默认短信应用的设置。
所以冒昧问一下能不能让NekoSMS不依赖xposed、直接设置为默认短信应用？
技术小白，Github账号也是为了提问专门注册的，如果问题很幼稚请不要介意。谢谢！

---

- Device: 索尼X1
- Android version: 11
- Xposed version: 无
- NekoSMS version: 无（之前在索尼Z1和XZ上用的是0.12.2和之后某个版本）

```
Replace this line with your logs. Do not remove the backticks.
```

Thank you for helping us help you help us all.

**Comment by apsun** [comment]
Hi, glad you enjoyed using NekoSMS! Unfortunately, the app doesn't have any SMS functionality other than showing blocked messages (it doesn't support sending, searching contacts, chats, etc, so it would be a pretty awful SMS app).

It would probably be easier to add the regex-style filtering functionality that NekoSMS has to an existing SMS app than it would be to add SMS functionality to NekoSMS. The idea of this app was always to be an "extension" to your favorite SMS app, not a replacement for it.

tl;dr: no, and there are no plans to add this, sorry.

**Comment by sammlicke** [comment]
Sorry to hear that but thank you all the same!
:)
